src/config/config.rs

```diff
@@ -112,10 +112,13 @@ impl Config {
 pub struct Type {
   pub fields: BTreeMap<String, Field>,
   pub doc: Option<String>,
+  #[serde(default)]
   pub interface: Option<bool>,
```

Comment:
```suggestion
  pub interface: bool,
```

We need to make a change like this everywhere.